Thang_LD/kiemtra.php

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Bai kiem tra</title>
+</head>
+<body>
+<?php
+		//bai 1.
+		//data
+
+		$fresherphp = array(
+	[
+		'first_name'=>'thieu',
+		'last_name' => 'nguyen',
+		'middle_name' => 'van',
+		'brith_day' => '01-06-1991',
+		'phone_number' => '01649593418'
+	],
+	[
+		'first_name'=>'thieu1',
+		'last_name' => 'nguyen1',
+		'middle_name' => 'van1',
+		'brith_day' => '01-06-1992',
+		'phone_number' => '0909593418'
+	],
+	[
+		'first_name'=>'thang',
+		'last_name' => 'luong',
+		'middle_name' => 'dinh',
+		'brith_day' => '01-06-1990',
+		'phone_number' => '0976149492'
+	]
+	);
+
+	// class sinh vien
+
+	class  Sinhvien
+	{
+		public $first_name;
+		public $last_name;
+		public $middle_name;
+		public $brith_day;
+		public $phone_number;
+		public $array;
+		
+		function __construct()
+		{
+		}
+		// mang chua thong tin sinh vien do.
+		function data_sinhvien($val){
+			$array = $val;
+			return $array;
+	    }
+
+	    #############
+	    // thong tin sinh vien
+
+
+	    function property($val)
+	    {
+	    	$first_name = $val['first_name'];
```

Comment:
Nếu sử dụng property thì em thực hiện gán giá trị như sau: `$this->first_name = $val['first_name'];`